api/rosetta/retriever.go

```diff
@@ -27,4 +27,5 @@ type Retriever interface {
 	Block(rosBlockID identifier.Block) (*object.Block, []identifier.Transaction, error)
 	Transaction(rosBlockID identifier.Block, rosTxID identifier.Transaction) (*object.Transaction, error)
 	Balances(rosBlockID identifier.Block, rosAccountID identifier.Account, rosCurrencies []identifier.Currency) (identifier.Block, []object.Amount, error)
+	AccountSequenceNumber(rosAccountID identifier.Account, keyIndex int) (uint64, error)
```

Comment:
`SequenceNumber` is enough. We already provide an account as input, so of course it's for an account. Actually, it's for an account and a key index. So why not call it `AccountKeySequenceNumber`? Because it's not necessary :D